mmif/serialize/mmif.py

```diff
@@ -511,7 +511,57 @@ def get_view_contains(self, at_types: Union[ThingTypesBase, str, List[Union[str,
                 if at_types in view.metadata.contains:
                     return view
         return None
-    
+
+    def _is_in_time_between(self, start: int, end: int, annotation: Annotation) -> bool:
+        s, e = self.get_start(annotation), self.get_end(annotation)
+        if (s < start < e) or (s > start and e < end) or (s < end < e):
+            return True
+        else:
+            return False
```

Comment:
I added an argument like `time_unit: str = "milliseconds"` in the `get_annotations_between_time` function to let user choose the unit. And then the function would check this input unit with the time unit of `TimeFrame` in a `View`, if they are not the same, some conversions would be triggered otherwise no. Thus, we can make sure the unit is always the same.